### PR TITLE
Update README - Add Instructions To Upgrade SDK Version

### PR DESCRIPTION
**Pull request by agedd**
### Summary of changes

- Add information about upgrading to the latest SDK version based on how our SDK has been integrated.

### Checklist

- [ ]~Added a changelog entry~

- @agedd

### CODE REVIEW

**Review by stechiu — APPROVED** [review]
Looks good, thanks for the changes!

File: README.md [hunk 1]
```diff
@@ -69,6 +69,21 @@ Start with [**'Hello, Client!'**](https://developer.paypal.com/braintree/docs/st
 
 Next, read the [**full documentation**](https://developer.paypal.com/braintree/docs/guides/payment-method-types-overview) for information about integrating with additional payment methods, such as PayPal and Venmo, as well as explore our pre-built [Drop-In UI offering](https://developer.paypal.com/braintree/docs/guides/drop-in/overview).
 
+## Upgrade Your SDK Version
+
+If you're looking to update to a newer version of our SDK, please see our recommended approach below.
```
Comment (jaxdesmarais) [inline README.md:74]:
```suggestion
If you're looking to update to a newer version of our SDK, please see our recommended approach below for each package manager. We always recommend updating to the latest version of the SDK which can be found in our [CHANGELOG](https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md).
```

File: README.md [hunk 2]
```diff
@@ -69,6 +69,21 @@ Start with [**'Hello, Client!'**](https://developer.paypal.com/braintree/docs/st
...
+
+### Using Swift Package Manager
+
+ To update using SPM, select _File→ Packages → Update to Latest Package Versions_.
```
Comment (jaxdesmarais) [inline README.md:78]:
```suggestion
 To update using Swift Package Manager, select _File→ Packages → Update to Latest Package Versions_.
```

File: README.md [hunk 3]
```diff
@@ -69,6 +69,21 @@ Start with [**'Hello, Client!'**](https://developer.paypal.com/braintree/docs/st
...
+
+### Using Cocoapods 
+
+You can either update all pods listed within your Podfile using `pod update` or specific pods as needed using `pod update PODNAME`. For additional detail, see these [guidelines](https://guides.cocoapods.org/using/pod-install-vs-update.html).
```
Comment (jaxdesmarais) [inline README.md:82]:
```suggestion
You can either update all pods listed within your Podfile using `pod update` or specific pods as needed using `pod update PODNAME`. For additional details, see the [Cocoapods guidelines](https://guides.cocoapods.org/using/pod-install-vs-update.html).
```

File: README.md [hunk 4]
```diff
@@ -69,6 +69,21 @@ Start with [**'Hello, Client!'**](https://developer.paypal.com/braintree/docs/st
...
+
+### Using Carthage
+
+To update to the latest versions of each framework, simply run the `carthage update` command. For more details, check out these [guidelines](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement).
```
Comment (jaxdesmarais) [inline README.md:86]:
```suggestion
To update to the latest versions of each framework, simply run the `carthage update` command. For more details, check out the [Carthage guidelines](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement).
```

**Review by jaxdesmarais — APPROVED** [review]
Just some general cleanup suggestions, otherwise looks great!

**Pull request merged**